test/models/user_test.rb

```diff
@@ -4,4 +4,11 @@ class UserTest < ActiveSupport::TestCase
   # test "the truth" do
   #   assert true
   # end
+  def setup
+    @user = User.new(name: "Example User", email: "user@example.com")
+  end
+
+  test "should be valid" do
```

Comment:
[IMO]
when you add the test. We must include two case: `check valid` and `check invalid`
Could you add one more test case to cover it?